### PR TITLE
feat(icons): new icons may thru august

### PR DESCRIPTION
**Pull request by conradennis**
Submitting new and updated UI icons from May through August 2022
all updates and additions approved by BXD

59 net new icons
4 updated icons
1 deprecated icon
1 new category

updated categories.yml with additions and new category `Brand > Design and development`
updated ui-icon-master.ai with additions and updates
updated deprecated.yml with removed icon

#### Changelog

**Net New (59 total)**
– 32/build-tool.svg
– 32/catalog--publish.svg
– 32/continuous-deployment.svg
– 32/continuous-integration.svg
– 32/ibm--datastage.svg
– 32/ibm--mq.svg
– 32/ibm--power-vs.svg
– 32/ibm-cloud--citrix-daas.svg
– 32/ibm-cloud--direct-link-1--connect.svg
– 32/ibm-cloud--direct-link-1--dedicated-hosting.svg
– 32/ibm-cloud--direct-link-1--dedicated.svg
– 32/ibm-cloud--direct-link-1--exchange.svg
– 32/ibm-cloud--direct-link-2--connect.svg
– 32/ibm-cloud--direct-link-2--dedicated-hosting.svg
– 32/ibm-cloud--direct-link-2--dedicated.svg
– 32/ibm-cloud--event-notification.svg
– 32/ibm-cloud--event-streams.svg
– 32/ibm-cloud--hyper-protect-crypto-services.svg
– 32/ibm-cloud--hyper-protect-dbaas.svg
– 32/ibm-cloud--hyper-protect-vs.svg
– 32/ibm-cloud--key-protect.svg
– 32/ibm-cloud--logging.svg
– 32/ibm-cloud--mass-data-migration.svg
– 32/ibm-cloud--pal.svg
– 32/ibm-cloud--secrets-manager.svg
– 32/ibm-cloud--sysdig-secure.svg
– 32/ibm-cloud-pak--netezza.svg
– 32/ibm-watson--assistant.svg
– 32/ibm-watson--discovery.svg
– 32/ibm-watson--knowledge-catalog.svg
– 32/ibm-watson--knowledge-studio.svg
– 32/ibm-watson--language-translator.svg
– 32/ibm-watson--machine-learning.svg
– 32/ibm-watson--natural-language-classifier.svg
– 32/ibm-watson--natural-language-understanding.svg
– 32/ibm-watson--openscale.svg
– 32/ibm-watson--query.svg
– 32/ibm-watson--speech-to-text.svg
– 32/ibm-watson--studio.svg
– 32/ibm-watson--text-to-speech.svg
– 32/ibm-watson--tone-analyzer.svg
– 32/keyboard--off.svg
– 32/logo--angular.svg
– 32/logo--figma.svg
– 32/logo--invision.svg
– 32/logo--npm.svg
– 32/logo--react.svg
– 32/logo--sketch.svg
– 32/logo--svelte.svg
– 32/logo--vue.svg
– 32/test-tool.svg
– 32/promote.svg
– 32/repo--artifact.svg
– 32/repo--source-code.svg
– 32/side-panel--close--filled.svg
– 32/side-panel--close.svg
– 32/side-panel--open--filled.svg
– 32/side-panel--open.svg
– 32/unsaved.svg

**Updated (4 total)**
– 32/ibm-cloud--internet-services
– 32/logo--discord
– 32/logo--medium
– 32/tools--alt

**Moved Categories (7 total)**
– 32/logo--jupyter
– 32/logo--keybase
– 32/logo--kubernetes
– 32/logo--openshift
– 32/logo--python
– 32/logo--r-script
– 32/logo--vmware

**Deprecated (1 total)**
– 32/direct-link

### CODE REVIEW

File: packages/icons/deprecated.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 deprecated:
 - name: kubernetes
   reason: The current kubernetes icon has been replaced by logo--kubernetes
+- name: direct-link
+  reason: The current icon has been replaced with the correct version of the App icon
```
Comment (tay1orjones) [inline packages/icons/deprecated.yml]: Should `direct-link` no longer be used? If so, what should be used in it's place?

If this is only an artwork change and the name is staying the same, I don't believe we need to have it listed as a deprecation.

Comment (conradennis) [reply packages/icons/deprecated.yml]: @tay1orjones correct, `direct-link` should no longer be used. There's a whole suite of "direct link" related icons in this batch but the one that would be the 1:1 substitution is: `ibm-cloud--direct-link-2--dedicated`

Screen Shot 2022-09-06 at 2 54 31 PM

**Review by laurenmrice — APPROVED** [review]
✅ Looks good!

File: packages/icons/deprecated.yml (hunk 1 above)
Comment (abbeyhrt) [reply packages/icons/deprecated.yml]: Would it be possible to make a suggestion like that in the reason comment? Something like "The current icon has been replaced with the correct version of the App icon, use the most appropriate ibm-cloud--direct-link icon variant instead"

Comment (conradennis) [reply packages/icons/deprecated.yml]: @abbeyhrt yup, just updated the .yml file with your suggested update

**Review by tay1orjones — APPROVED** [review]

**Pull request merged**